checksums-buildspec.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.2
+
+env:
+  secrets-manager:
+    GITHUB_TOKEN: "github-eks-distro-pr-bot:github-token"
+
+phases:
+  pre_build:
+    commands:
+    - ./build/lib/setup.sh
+
+  build:
+    commands:
+    - echo "CHECKSUMS -text merge=union" > .gitattributes
```

Comment:
this is a git trick to force the rebase to always succeed for checksum files by just doing the union of the diff. This only comes into play for the source controller builds which build amd and arm separately and therefore end up changing the same file.